src/HaskellCI/GitHub.hs

```diff
@@ -349,7 +349,7 @@ makeGitHub _argv config@Config {..} prj jobs@JobVersions {..} = do
             { ghBranches = cfgOnlyBranches
             }
         , ghJobs = Map.singleton "linux" GitHubJob
-            { ghjName      = "Haskell-CI Linux"
+            { ghjName      = "Haskell-CI Linux - GHC ${{ matrix.ghc }}"
```

Comment:
Have you found a reference of that templating syntax anywhere, or are just copying the examples (that what I have done so far).

I wonder what happens when we want to add GHCJS support.

Reply:
> Have you found a reference of that templating syntax anywhere

Is [this](https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions) what you're looking for?

> I wonder what happens when we want to add GHCJS support.

In that case, I imagine that we'd need to switch from using `ghc: X.Y.Z` to something like this:

```yaml
- compiler-version: 8.10.2
  compiler: "GHC"
- compiler-version: 8.10.2
  compiler: "GHCJS"
```

And then change `name` to be something like `"Haskell-CI Linux - ${{matrix.compiler}} ${{matrix.compiler-version}}"`. Disclaimer: I haven't thought through the ramifications of this design thoroughly.